README.md

```diff
@@ -481,6 +483,26 @@ List of supported Datadog services:
     * `datadog_user`
 
 
+### Use with Cloudflare
+
+Example:
+```
+CLOUDFLARE_TOKEN=[CLOUDFLARE_API_TOKEN]
+CLOUDFLARE_EMAIL=[CLOUDFLARE_EMAIL]
+ ./terraformer import cloudflare --resources=firewall,dns
```

Comment:
If you want, you can add token and email by flag, look github example. Not must

Reply:
I thought about it and decided put creds in env vars for prettier command invocation